Makefile

```diff
@@ -3,7 +3,7 @@
 # To re-generate a bundle for another specific version without changing the standard setup, you can:
 # - use the VERSION as arg of the bundle target (e.g make bundle VERSION=0.0.2)
 # - use environment variables to overwrite this value (e.g export VERSION=0.0.2)
-VERSION ?= 2.5.0-beta.6
+VERSION ?= 2.7.0-beta.9
```

Comment:
Should this be `2.7.0`? 

I assume our release process probably overrides it (especially if it was 2.5.0-beta.6 beforehand).

Reply:
Fixed